### DISCUSSION TITLE
Käyttöhäiriö 14.8. noin klo 00.00 - 10.15

### DISCUSSION THREAD

**Discussion by CodoDerDritte** [KohaSuomi/Koha#1353] [Huolto- ja häiriötiedotteet]
Koha palvelimilla oli tallennusjärjestelmän toimintahäiriö keskiviikkona 14.8. klo 00.00 - 10.15. Kaikki Kohat olivat tuolla aikavälillä saavuttamattomissa.

Häiriöstä johtuen menetettiin jonkin verran indeksi- ja lokitietoja. Lisäksi yölle ajastetut tehtävät ovat jääneet ajamatta. Ajastetut tehtävät ajetaan manuaalisesti tämän päivän aikana. Tällaisia ovat esimerkiksi asiakkaan maksujen laskenta, asiakastyyppimuutokset, varausten keskeytysten käsittely ym. Indeksit päivitetään tuotantoympäristöissä ensi yönä, joten haku on täysin toiminnassa huomenna aamulla. Tämän päivän joudumme siis ikävä kyllä menemään hieman puutteellisilla indekseillä, joten hakuihin kannattaa mahdollisuuksien mukaan käyttää Finna-hakua sekä Kohan nidehakua, jota indeksiongelma ei koske.

Käytettävissä ovat tällä hetkellä tuotanto- ja next-ympäristöt. Testiympäristöjä ei ole vielä käynnistetty häiriön jäljiltä uudelleen. Tiedotamme tässä ketjussa kun testiympäristötkin ovat taas käytössä ja mikäli muuta uutta tiedotettavaa ilmenee.

Pahoittelemme häiriötä.

**Comment by johannaraisa** [comment]
Puuttuvien croniajojen uudelleen ajamista voi seurata tiketissä [1356](https://github.com/KohaSuomi/Koha/issues/1356)

**Comment by CodoDerDritte** [comment]
Jäkipyykki jatkuu kakkos- ja nelosnodeilla (Jo ja Amy), joten testit ja nextit saattavat joko toimia tai olla toimimatta riippuen siitä milloin kokeilee. Tuotannoissa häiriöitä ei pitäisi ilmetä.
